src/engine/enginebuffer.cpp

```diff
@@ -185,6 +185,9 @@ EngineBuffer::EngineBuffer(QString group, UserSettingsPointer pConfig,
             this, SLOT(slotEjectTrack(double)),
             Qt::DirectConnection);
 
+    m_pTrackLoaded = new ControlObject(ConfigKey(m_group, "track_loaded"), 0, 1);
```

Comment:
Right, I'm asking why you're passing values for the 2nd and 3rd args here (2nd arg is ignore-nops, and 3rd arg is to enable tracking). Maybe they're left over from it being a potmeter or something?